pkg/attestlib/ec_sign.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package attestlib
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"encoding/asn1"
+	"github.com/pkg/errors"
+	"math/big"
+)
+
+func ecSign(privateKey *ecdsa.PrivateKey, payload []byte, signatureAlgorithm SignatureAlgorithm) ([]byte, error) {
```

Comment:
Document the format of the returned signature

Reply:
done.